packages/ui/src/molecules/Modal/Modal.tsx

```diff
@@ -44,6 +52,8 @@ const Modal = ({
   children,
   onDismiss,
   testId = 'store-modal',
+  overlayId,
+  className,
```

Comment:
Missed this one!

```suggestion
  overlayClassName,
```

Reply:
Why need this? We style html elements via data attributes